packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/routing/layout/layout.component.html

```diff
@@ -0,0 +1,37 @@
+<app-navigation></app-navigation>
+<div class="{{ mainClassPageEditing }}">
+  <ng-container *ngIf="state === LayoutState.Layout">
+    <app-scripts></app-scripts>
+    <header>
+      <div id="header">    
+        <sc-placeholder
+          name="headless-header"
+          [rendering]="route"
+        ></sc-placeholder>
+      </div>
+    </header>
+    <main>
+      <div id="content">
+        <sc-placeholder
+          name="headless-main"
+          [rendering]="route"
+          (loaded)="onMainPlaceholderLoaded($event)"
```

Comment:
Rename loaded function
```suggestion
          (loaded)="onPlaceholderLoaded($event)"
```